task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -72,13 +72,15 @@ spec:
           popd
           exit 0
         fi
+        # copy content of conffolder to confdir - will be used in next task - related image check
+        cp -r .$conffolder/* $(workspaces.workspace.path)/hacbs/$(context.task.name)/confdir
 
         # We have totally 4 checks here currently
         check_num=4
         failure_num=0
         TESTPASSED=true
 
-        if [ ! $(find . -name "opm") ]; then
+        if [[ ! $(find . -name "opm") ]]; then
```

Comment:
few lines bellow, is there the same issue with single `[` ?

Also few lines bellow
```
if ! opm validate ."${conffolder}"; then
```

Can we unify how to check return code?

Reply:
Its not raising the issue for other lines, let me try to unify.

Reply:
Tried to unify and failed miserably, take a look.

Reply:
Unfortunately this seems like a way to go.